config_syntax.md

```diff
@@ -174,15 +174,74 @@ _Required:_ no
 
 _Default:_ False
 
+### http_address
+**The address to listen on for new web connections
 
+_Type:_ String
+
+_Required:_ no
+
+_Default:_ 127.0.0.1
+
+### http_port
+**The port to listen on for new web connections
+
+_Type:_ Integer
+
+_Required:_ no
+
+_Default:_ 8080
+
+### use_https
+**The port to listen on for new web connections
+
+_Type:_ Boolean
+
+_Required:_ no
+
+_Default:_ False
+
+### https_key_file
+**If using HTTPS, this specifies where the private key file is located
+
+_Type:_ String
+
+_Required:_ no
+
+_Default:_ ./key.pem
+
+### https_key_file
+**If using HTTPS, this specifies where the certificate file is located
+
+_Type:_ String
+
+_Required:_ no
+
+_Default:_ ./cert.pem
+
+The webserver is provided principally to allow polling of the winlink_monitor for its
```

Comment:
Bring up above the options for it?

Reply:
And add docs on the paths (basically the help)?

Reply:
Done.